ert_shared/storage/storage_api.py

```diff
@@ -17,6 +17,22 @@ def __init__(self, rdb_api=None, blob_api=None):
         else:
             self._blob_api = blob_api
 
+    def _ensemble_minimal(self, ensemble):
+        return {
+            "name": ensemble.name,
+            "ensemble_ref": ensemble.id,
+            "parent": {
+                "ensemble_ref": ensemble.parent[0].ensemble_reference.id,
+                "name": ensemble.parent[0].ensemble_reference.name
```

Comment:
This seems wrong. There is a list of parents, but there should be a one-to-one relationship, probably